.ci/install_nemu.sh

```diff
@@ -19,21 +19,23 @@ arch=$("${cidir}"/kata-arch.sh -d)
 install_nemu() {
 	local nemu_repo=$(get_version "assets.hypervisor.nemu.url")
 	local nemu_version=$(get_version "assets.hypervisor.nemu.version")
+	PACKAGING_REPO="github.com/kata-containers/packaging"
+	NEMU_TAR="kata-nemu-static.tar.gz"
 	case "$arch" in
 	x86_64)
-		local nemu_bin="qemu-system-${arch}_virt"
-		;;
-	aarch64)
```

Comment:
Why is aarch64 being dropped? Even if we don't provide the static binaries today, NEMU works on arm64 too so can't we leave it as we should endeavor to add test support for that arch at some future point too?

Reply:
We can add aarch64 here back, but it does come with some limitations. 1. It is not statically built, so we will need to add more scaffolding to the install scripts to get all the dependent libraries for it to work. It will also only work with xenial and probably fail on other distros. In the current form, it is not really consumable. As stage 2, we can start shipping `aarch64` static binaries also along with `x86_64` and then re-enable it. I can file a bug for that, if that approach would work.

Reply:
@Pennyzct @justin-he Is this something that you could help with? We really would like to see support for building nemu on arm.
@ganeshmaharaj Can you open a separate issue for this, and list out what issues you are running into there?

Reply:
https://github.com/kata-containers/tests/issues/1620